Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_fd.nut

```diff
@@ -1043,6 +1055,63 @@ void function FD_createHarvester()
 	fd_harvester.harvester.Minimap_SetZOrder( MINIMAP_Z_OBJECT )
 	fd_harvester.harvester.Minimap_SetCustomState( eMinimapObject_prop_script.FD_HARVESTER )
 	AddEntityCallback_OnDamaged( fd_harvester.harvester, OnHarvesterDamaged )
+	thread CreateHarvesterHintTrigger( fd_harvester.harvester )
+}
+
+void function CreateHarvesterHintTrigger( entity harvester )
+{
+	entity trig = CreateEntity( "trigger_cylinder" )
+	trig.SetRadius( 1000 )	//Test setting
+	trig.SetAboveHeight( 2500 )	//Test setting
+	trig.SetBelowHeight( 0 )	//Test setting
```

Comment:
```suggestion
	trig.SetBelowHeight( 2500 )	//Test setting
```
Probably worth extending this below as well, just in case of maps where the harvester is on top of something